File: Go/two.go
package main

import "fmt"

func fib_sum_even(limit int) int {
	var (
		sum int = 0
		a   int = 1
		b   int = 2
	)

	for b < limit {
		if b%2 == 0 {
			sum += b
		}
		a, b = b, a+b
	}
	return sum
}

func main() {
	sum := fib_sum_even(4000000)
	fmt.Println(sum)
}

File: Python/two.py
def fib_sum_even(limit: int) -> int:
    '''
    finds sum of even terms in fibonacci sequence upto limit.
    '''
    a = 1
    b = 2
    sum = 0

    while b < limit:
        if b % 2 == 0:
            sum += b
        a, b = b, a+b
    return sum


if __name__ == "__main__":
    print(fib_sum_even(4000000))
